sdk/python/feast/diff/registry_diff.py

```diff
@@ -8,16 +8,14 @@
 from feast.feast_object import FeastObject, FeastObjectSpecProto
 from feast.feature_service import FeatureService
 from feast.feature_view import DUMMY_ENTITY_NAME
-from feast.protos.feast.core.DataSource_pb2 import DataSource as DataSourceProto
-from feast.protos.feast.core.Entity_pb2 import Entity as EntityProto
-from feast.protos.feast.core.FeatureService_pb2 import (
-    FeatureService as FeatureServiceProto,
-)
-from feast.protos.feast.core.FeatureView_pb2 import FeatureView as FeatureViewProto
-from feast.protos.feast.core.OnDemandFeatureView_pb2 import (
+from feast.proto_core.DataSource_pb2 import DataSource as DataSourceProto
```

Comment:
I am a little bit unclear on what problem you are trying to solve with this change? Didn't we have absolute imports before and after?

Reply:
While we did, there was a bug that I found later and fixed in #2437.

With this PR - I just think that the subdir structure is a bit stuttery - `feast.protos.feast.*`. It's a taste thing, not a major functionality thing.